mace/cli/fine_tuning_select.py

```diff
@@ -55,17 +63,33 @@ class SelectionSettings:
     weight_pt: float = 1.0
     allow_random_padding: bool = True
     seed: int = 42
+    config: str | None = None
 
 
 def str_to_list(s: str) -> List[int]:
     assert isinstance(s, str), "Input must be a string"
     return ast.literal_eval(s)
 
 
-def parse_args() -> argparse.Namespace:
-    parser = argparse.ArgumentParser(
-        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
-    )
+def build_default_finetuning_select_arg_parser() -> argparse.ArgumentParser:
```

Comment:
I have a suggestion that would help merge this with doing filtering in `mace_run_train`.  Have this function get an optional parser argument, and a suffix.  If the parser argument is passed, the options (I guess not `--config`) are added to that one, otherwise it creates a new parser object.  It also adds the suffix to all the arguments, so that `mace_run_train` can pass `suffix=_mp` and things like `--filter_type`, which makes sense for `mace_finetune_select`, will become `--filter_type_mp`, the way they are now in the `develop` branch.

Reply:
I just noticed, above, that this won't work right now because this code also handles fitting data at the same time. I'd like to suggest going through all the command line arguments and making them as much like `mace_run_train` as possible.